docs/setup.md

```diff
@@ -3,43 +3,50 @@
 
 # Installation
 
+This widget automatically installs with the **Outbreak Response Framework** solution pack.
+
```

Comment:
This widget will be installed as part of SP only same as CICD. Please check CICD doc. IMO, your changes are not needed.